scripts/build-package

```diff
@@ -3,6 +3,10 @@ set -e
 
 pip install --require-hashes -r requirements.txt --target ./package
 pip install --no-deps --target ./package .
+git clone git+https://${GITHUB_TOKEN}@github.com/release-engineering/cdn-definitions-private.git
+./cdn-definitions-private/scripts/make-json
```

Comment:
depends on https://gitlab.cee.redhat.com/cdn/cdn-definitions-private/-/merge_requests/6

Reply:
I don't think we should do this:

The approach being proposed here, of having cdn-definitions-private only keep a YAML file and then running a script to produce a JSON file, doesn't make sense to me.  I've written a bigger comment at https://github.com/release-engineering/cdn-definitions/pull/19/files#r568302386 about YAML vs JSON.

I think it would make sense to focus on completion of the cdn-definitions change first and then come back to this once that's been released.